divvy/cloudfilebrowser.scss

```diff
@@ -405,6 +405,44 @@ the License.
     }
 
     #services-containers > div.provisioned > a.provision { display: none }
+
+    #services-containers > div.provisioned > div.search-wrapper {
```

Comment:
Without looking too much into this CSS, I just want to make sure you reused as much of the styling as we already have in Divvy 3.0.

Reply:
Reused sir.